scss/variables/notifications.js

```diff
@@ -0,0 +1,70 @@
+( function( root, factory ) {
+	if ( typeof define === "function" && define.amd ) {
+		define( [ "./chassis" ], factory );
+	} else if ( typeof exports === "object" ) {
+		module.exports = factory( require( "./chassis" ) );
+	} else {
+		root.chassis = factory( root.chassis );
+	}
+}( this, function( chassis ) {
+
+chassis.notifications = {
+
+	"shadows": {
+		name: "Notification Shadow Style",
+		value: {
+			"box-shadow-base": "0px 8px 11px 0px rgba(0,0,0,0.15)"
+		}
+	},
+	"element": {
+		name: "Generic Notification Style",
+		value: {
+			"border-radius": "3px",
+			"margin": ".25em",
+			"letter-spacing": "1px",
+			"font-weight": "400",
+			"font-size": "16px",
+			"padding": "15px 15px 15px 25px"
+		}
+	},
+	"default": {
+		name: "Default Notification",
+		value: {
+
+			// Need help here on setting background color
+			"color": "#ffffff",
+			"background": "#333333"
+		}
+	},
+	"info": {
+		name: "Info Notification",
+		value: {
+			"color": "#ffffff",
+			"background": "#4882B7"
```

Comment:
Fails all Contrast tests for normal size text. Need more contrast in color, but that can wait, and will be fixed automatically if you use palette  colors